lua/entities/gmod_wire_expression2/core/debug.lua

```diff
@@ -420,9 +420,12 @@ end
 util.AddNetworkString( "wire_expression2_set_clipboard_text" )
 local clipboard_character_limit = CreateConVar("wire_expression2_clipboard_character_limit", 512, FCVAR_ARCHIVE, "Maximum character that can be copied into a players clipboard", 0, 65532)
 local clipboard_cooldown = CreateConVar("wire_expression2_clipboard_cooldown", 1, FCVAR_ARCHIVE, "Cooldown for setClipboardText in seconds", 0, nil)
+local clipboard_allow_toggle = CreateConVar("wire_expression2_clipboard_allow", 0, FCVAR_ARCHIVE, "Cooldown for setClipboardText in seconds", 0, 1)
 
 __e2setcost(100)
 e2function void setClipboardText(string text)
+	if clipboard_allow_toggle:GetInt() == 0 then return self:throw("setClipboardText is disabled by this server", nil) end
```

Comment:
Could use `:GetBool()` instead, and you don't *need* to limit the range of the convar in this case, but these are just minor nitpicks.